gatsby-config.js

```diff
@@ -71,16 +71,16 @@ const plugins = [
             },
           },
         },
-        {
-          resolve: `gatsby-remark-vscode`,
-          options: {
-            theme: "Dark+ (default dark)",
-            languageAliases: {
-              solidity: "js",
-              sol: "js",
-            },
-          },
-        },
+        // {
+        //   resolve: `gatsby-remark-vscode`,
+        //   options: {
+        //     theme: "Dark+ (default dark)",
+        //     languageAliases: {
+        //       solidity: "js",
+        //       sol: "js",
+        //     },
+        //   },
+        // },
```

Comment:
Let's uncomment this part. I'm guessing it wasn't meant to be included in this PR.